geometry/create_capacitance_file_theta.py

```diff
@@ -0,0 +1,381 @@
+# calculate and save histograms of capacitance per source vs theta
+# output is saved in ROOT file with given filename
+#
+# execute script with
+# python create_capacitance_file_theta.py
+#
+
+from ROOT import TH1F, TF1, TF2, TCanvas, TLegend, TFile, gStyle
+import ROOT
+from math import ceil, sin, cos, atan, exp, log, tan, pi, sqrt, asin, degrees, radians
+
+ROOT.gROOT.SetBatch(ROOT.kTRUE)
+
+gStyle.SetPadTickY(1)
+
+debug = False
+verbose = True
+
+#Dimensions
+### FCCee
+#ECalConstruction     INFO ECAL cryostat: front: rmin (cm) = 210 rmax (cm) = 215 dz (cm) = 226
+#ECalConstruction     INFO ECAL cryostat: back: rmin (cm) = 260 rmax (cm) = 270 dz (cm) = 226
+#ECalConstruction     INFO ECAL cryostat: side: rmin (cm) = 215 rmax (cm) = 260 dz (cm) = 10
+#ECalConstruction     INFO ECAL services: front: rmin (cm) = 215 rmax (cm) = 216 dz (cm) = 216
+#ECalConstruction     INFO ECAL services: back: rmin (cm) = 256 rmax (cm) = 260 dz (cm) = 216
+#ECalConstruction     INFO ECAL bath: material = LAr rmin (cm) =  216 rmax (cm) = 256 thickness in front of ECal (cm) = 1 thickness behind ECal (cm) = 4
+#ECalConstruction     INFO ECAL calorimeter volume rmin (cm) =  216 rmax (cm) = 256
+#ECalConstruction     INFO passive inner material = Lead and outer material = lArCaloSteel thickness of inner part (cm) =  0.14 thickness of outer part (cm) =  0.04 thickness of total (cm) =  0.2 rotation angle = 0.872665
+#ECalConstruction     INFO number of passive plates = 1536 azim. angle difference =  0.00409062
+#ECalConstruction     INFO  distance at inner radius (cm) = 0.883573 distance at outer radius (cm) = 1.0472
+#ECalConstruction     INFO readout material = PCB thickness of readout planes (cm) =  0.12 number of readout layers = 12
+#ECalConstruction     INFO thickness of calorimeter (cm) = 40 length of passive or readout planes (cm) =  56.4964
+#ECalConstruction     INFO active material = LAr active layers thickness at inner radius (cm) = 0.247949 thickness at outer radious (cm) = 0.479746 making 93.4857 % increase.
+#ECalConstruction     INFO active passive initial overlap (before subtraction) (cm) = 0.1 = 50 %
+
```

Comment:
Should be removed if it does not correspond to the current geometry

Reply:
done